assets/src/components/AssignmentTable.js

```diff
@@ -137,6 +137,7 @@ function AssignmentTable (props) {
   const [previousWeek, setPreviousWeek] = useState()
 
   const weeksPresent = useRef([])
+  const scrollOnce = useRef(false)
```

Comment:
Was thinking about this and it may make more sense to call it shouldScrollToCurrentWeek or something so its more clear what the use is rather than the function.

That way if we decided some action should warrant scrolling more than once per page load we can turn it back on and it makes more sense than setting scrollOnce to false again.